casper/src/test/scala/coop/rchain/casper/engine/LFSBlockRequesterEffectsSpec.scala

```diff
@@ -0,0 +1,276 @@
+package coop.rchain.casper.engine
+
+import cats.effect.{Concurrent, Sync}
+import cats.syntax.all._
+import com.google.protobuf.ByteString
+import com.sun.tools.javac.code.Types.SignatureGenerator.InvalidSignatureException
+import coop.rchain.casper.protocol._
+import coop.rchain.casper.util.TestTime
+import coop.rchain.models.BlockHash.BlockHash
+import coop.rchain.shared.{Log, Time}
+import fs2.Stream
+import fs2.concurrent.Queue
+import monix.eval.Task
+import monix.execution.atomic.AtomicAny
+import org.scalatest.prop.GeneratorDrivenPropertyChecks
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.concurrent.duration._
+
+class LFSBlockRequesterEffectsSpec
+    extends FlatSpec
+    with Matchers
+    with GeneratorDrivenPropertyChecks {
+
+  def blockHash(x: Byte): (BlockMessage => BlockHash) = _ => ByteString.copyFrom(Array[Byte](x))
+
+  def mkHash(bs: Byte*) = ByteString.copyFrom(bs.toArray)
+
+  def getBlock(hash: BlockHash, number: Long, latestMessages: Seq[BlockHash]) =
```

Comment:
Can you use https://github.com/rchain/rchain/blob/ede635ed33cbbccf60e9a47296b4a8910eb818e7/models/src/test/scala/coop/rchain/models/blockImplicits.scala#L212-L226 here?